repose-aggregator/components/filters/versioning/pom.xml

```diff
@@ -63,14 +63,20 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <scope>test</scope>
+        </dependency>
         <dependency>
-            <groupId>org.codehaus.jackson</groupId>
-            <artifactId>jackson-core-lgpl</artifactId>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
             <scope>test</scope>
```

Comment:
I couldn't find a definitive answer on this one way or the other. There was just a bunch of folks saying "I think it ...". So the final test was to run`mvn dependency:tree`. ;-)